code/modules/mob/living/carbon/carbon.dm

```diff
@@ -89,6 +89,8 @@
 
 
 /mob/living/carbon/electrocute_act(shock_damage, obj/source, siemens_coeff = 1, override = 0, tesla_shock = 0)
+	if(dna && dna.check_mutation(SMES_HUMAN))

```

Comment:
All species have siemens coefficient that is like that on gloves
Just change it to 0 on mut_gain and to initial on mut_lose